templates/flutter/pubspec.yaml.twig

```diff
@@ -13,15 +13,15 @@ platforms:
   web:
   windows:
 environment:
-  sdk: '>=2.17.0 <3.0.0'
+  sdk: '>=2.17.0 <4.0.0'
```

Comment:
If you want me to revert this change and also the `http` constraints change, please let me know and I will remove these from this PR.